dts/arm/st/stm32l4.dtsi

```diff
@@ -285,6 +285,16 @@
 			prop_seg_phase_seg1 = <5>;
 			phase_seg2 = <6>;
 		};
+
+		rtc: rtc@40002800 {
```

Comment:
The rtc node property `prescaler` is missing. It is defined as required in rtc.yaml.

Reply:
Not sure what its semantics would be for this driver.

Reply:
If it is not necessary for all SoCs, we should make it optional in rtc.yaml.